examples/chip-tool/commands/discover/Commands.h

```diff
@@ -18,38 +18,23 @@
 
 #pragma once
 
-#include "Command.h"
+#include "DiscoverCommand.h"
+#include <controller/DeviceAddressUpdater.h>
 #include <mdns/Resolver.h>
 
-class Discover : public Command, public chip::Mdns::ResolverDelegate
+class Resolve : public DiscoverCommand, public chip::Mdns::ResolverDelegate
 {
 public:
-    Discover() : Command("resolve-node-id")
-    {
-        AddArgument("nodeid", 0, UINT64_MAX, &mNodeId);
-        AddArgument("fabricid", 0, UINT64_MAX, &mFabricId);
-    }
+    Resolve() : DiscoverCommand("resolve") {}
 
-    CHIP_ERROR Run(PersistentStorage & storage, NodeId localId, NodeId remoteId) override
+    /////////// DiscoverCommand Interface /////////
+    CHIP_ERROR RunCommand(NodeId remoteId, uint64_t fabricId) override
```

Comment:
Would a better name for this simply be `Resolve`, or perhaps `Discover`?

Reply:
Pretty sure it's this way as it follows the current RunCommand/Run style of the class (Resolve).